tests/test__default.py

```diff
@@ -540,6 +541,29 @@ def test__get_explicit_environ_credentials_no_env():
     assert _default._get_explicit_environ_credentials() == (None, None)
 
 
+def test_load_credentials_from_file_external_account_authorized_user(tmpdir):
+    config_file = tmpdir.join("config.json")
```

Comment:
You can create a file under data folder so that this config can be reused later. We already have few jsons there

Reply:
Done